opt_maven_install.sh

```diff
@@ -21,7 +21,7 @@ if [[ "$1" == "true" ]]; then
     # build presto
     cd /build && mvn --batch-mode --errors -Dmaven.javadoc.skip=true -Dmaven.source.skip=true -DskipTests -DfailIfNoTests=false -Dtest=false -DdownloadSources -DdownloadJavadoc clean package -pl '!presto-testing-docker' -Dmaven.repo.local=.m2/repository
     # Install prometheus-jmx agent
-    mvn --batch-mode dependency:get -Dartifact=io.prometheus.jmx:jmx_prometheus_javaagent:0.3.1:jar -Ddest=/build/jmx_prometheus_javaagent.jar
+    mvn --batch-mode dependency:get -Dartifact=io.prometheus.jmx:jmx_prometheus_javaagent:0.3.1:jar -Ddest=/build/jmx_prometheus_javaagent.jar && mv $HOME/.m2/repository/io/prometheus/jmx/jmx_prometheus_javaagent/0.3.1/jmx_prometheus_javaagent-0.3.1.jar /build/jmx_prometheus_javaagent.jar
```

Comment:
I think there is some slicker Maven way to do this, but I appreciate the simplicity.

Reply:
Yeah absolutely - just trying to unblock us for now, but we should really invest some time into making sure our maven environment is properly set up (for both this and hadoop) so we don't have to do this sort of thing.